### PR TITLE
[TS] LPS-199171

### PR DESCRIPTION
**Pull request by yuhai**
@dantewang, the previous pr is https://github.com/liferay-core-infra/liferay-portal/pull/5856. LPS-68101 (https://github.com/liferay-core-infra/liferay-portal/commit/c5c3c849624f6e41009d8d2b27e8eaacd9d702d7) was backported to 7.0.x. This will exist duplicated codes when building 7.0 version with including LPS-68101 fix. We didn't find a good way to avoid it yet.

cc @marianoalvarosaiz

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
@yuhai Found no regressions. But I tried running this on a few services in 7.0 and this change isn't applied. I don't see any services that use version 7_1_0 in 7.0. Is this expected? Just want to clarify before passing this.

**Comment by marianoalvarosaiz** [comment]
> @yuhai Found no regressions. But I tried running this on a few services in 7.0 and this change isn't applied. I don't see any services that use version 7_1_0 in 7.0. Is this expected? Just want to clarify before passing this.

No services should use 7_1_0 in 7.0, but the fix should apply to fixes that use 7_0_0 in 7.0 because it checks that the DTD is not equals or greater than 7_1_0

Does the fix still doesn't apply to any service? How did you test it to try to reproduce it?

**Comment by Kvale1733** [comment]
marianoalvarosaiz Oh I see. I build the jar on master an used it on 7.0. I tested on a few services like journal-service to see if the changes applied but they didn't. I'll try again to make sure.

**Comment by Kvale1733** [comment]
Tested this on 7.0 with document-library-service and changes were applied. No regressions found.

**Comment by yuhai** [comment]
@Kvale1733, thanks for your test.

File: modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl [hunk 1]
```diff
@@ -251,6 +257,10 @@ public class ${entity.name}PersistenceImpl extends BasePersistenceImpl<${entity.
 		<#assign columnBitmaskEnabled = (entity.finderEntityColumns?size &gt; 0) && (entity.finderEntityColumns?size &lt; 64) && !entity.hasEagerBlobColumn() />
 	</#if>
 
+	<#if !serviceBuilder.isVersionGTE_7_1_0()>
```
Comment (dantewang) [inline modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl:260]: @yuhai We don't have a method checking if the version is less than 7.1.0, or less than or equals 7.0.0, right?

Comment (yuhai) [reply modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_impl.ftl:260]: @dantewang, yes, we don't have the method to check.

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/143153

**Pull request closed without merging**